bin/templates/cordova/lib/prepare.js

```diff
@@ -317,28 +318,13 @@ function getAdaptiveImageResourcePath (resourcesDir, type, density, name, source
     return resourcePath;
 }
 
-/**
- * The splashScreenSrc is a .webp image or a .png (or .jpg) image.
- * An inputted .webp should be copied to a screen.webp file.
- * An inputted .png (or .jpg) should be copied to a screen.png file.
- *
- * When a splashscreen is changed from a .png to a .webp or the other way round,
- * the other image must be deleted, to avoid a duplicate resource exception.
- *
- * @param splashScreenSrc
- * @returns {{splashScreenDestToRemove: string, splashScreenDest: string}}
- */
-function buildSplashScreenImageToCopyAndToRemove (splashScreenSrc) {
-    if (path.extname(splashScreenSrc) === '.webp') {
-        return {
-            splashScreenDest: 'screen.webp',
-            splashScreenDestToRemove: 'screen.png'
-        };
-    }
-    return {
-        splashScreenDest: 'screen.png',
-        splashScreenDestToRemove: 'screen.webp'
-    };
+function makeSplashCleanupMap (projectRoot, resourcesDir) {
+    // Build an initial resource map that deletes all existing splash screens
+    const existingSplashPaths = glob.sync(
```

Comment:
Seems like a forward and backwards slash problem in my tests. The resourcesDir is with backward slashes, but should be with forward slashes.

Reply:
Fixed in next commit.